scripts/storage-emulator-integration/run.sh

```diff
@@ -7,6 +7,8 @@ set -e # Immediately exit on failure
 # Prepare the storage emulator rules runtime
 firebase setup:emulators:storage
 
+mocha scripts/storage-emulator-integration/rules/*.test.ts
```

Comment:
I'm a little surprised you don't need the `-require` lines below, but it does look like the tests are being run (successfully, too).

Reply:
I believe mocha automatically pickes up .mocharc in the parent dir.

Reply:
That makes total sense. Probably means I could clean up a lot of these lines in the integration tests haha

Reply:
Done; PR here: #4312